getting-started/index.mkd

```diff
@@ -10,32 +10,4 @@ redirect_from:
     <h1>Getting Started</h1>
 </div>
 
-So, what are you looking to do?
```

Comment:
@rathim @sherryren I think it would be better to leave this section in. We could rewrite it a bit and reorder the icons to better suite the new layout?